defaults/main.yml

```diff
@@ -33,3 +33,17 @@ riemann_slack_tag: slack_alert
 
 # Disable verbose logging, useful for standing up the server.
 riemann_debug_logging: false
+
+riemann_create_folders:
+  - "conf.d"
+  - "utils"
```

Comment:
Good question buddy!!! Initially it was for technical reasons when I started off --- take a look at our riemann configs repo at the `scripts/pre-sync-templates.py` file. 

I had to do some tomfoolery in order for the templated riemann configs to be jinja-fied before I could run the local docker tests. It seemed more straight-forward logic to logically put all non-template files in `conf.d` and then all jinja stuff in `utils` ... at least that was the logic I had in my head...

We could just dump them all in conf.d ... would also have to go back and mess with that script though....